### ISSUE TITLE
request for raw data

### ISSUE THREAD

**Issue by midnightradio** [junkwhinger/overlog#1]
Hi,

could you please provide the raw data before preprocessing? Particularly, the "skillRating" that I need has been removed after producing "user_class" but I cannot regenerate the field without knowing its distribution. Please be generous in letting me proceed my own study which is in different context from yours.

**Comment by junkwhinger** [comment]
Hi there! Sorry for such a belated reply :)

Here's my original data in pickle format. Hope it helps!
[overlog_raw.pkl.zip](https://github.com/junkwhinger/overlog/files/644379/overlog_raw.pkl.zip)

**Comment by midnightradio** [comment]
OK, thanks for providing the data and I would like to show my appreciation to your openness as well. 

And I guess now is the right time to mention that related work has some problems which I found at a glance. Apart from some minors, you made a big mistake in evaluation on the model performance. You better put more care on reading the metric given by sklearn. Hope you can correct your mistake and find a way to improve your work. Maybe you can discuss with your colleges if they could notice the problem.

GOOD LUCK!

**Comment by junkwhinger** [comment]
Thank you for reviewing my work! As you have kindly pointed out I think there is a room or are rooms for improvements to make better performing models. I'd appreciate if you could share the link of your work in the near future :) guess I can get some useful hints and tips from you. Again, thank you for your review. Ciao

**Comment by midnightradio** [comment]
I'm sorry but let me go straight into the point as it seems that you are
not gonna properly understand my words. It's not about improving the
performance but I was pointing out a mistake you made for the evaluation.
You cannot just make an average like you did (actually it is done by
sklearn, which means you need to put more care on using them in a proper
way.), especially when the sample space is highly imbalanced. See how your
model poorly performing on minor class while it shows good accuracy on
majority. What you did is just summing them up and hiding poor performance
on minority by running weighted average on the scores.

Can't believe nobody have pointed out your mistake after you have
passionately shared your work. I'm really sorry to be the first and the
only one, but I'm sure my advice will do good to you in a long run. ;-)

Best regards,
Hongjoo Lee

**Comment by junkwhinger** [comment]
Thanks for your comment. Well what I implied earlier was that if my work had had some problems with model evaluation then it would definitely be possible to make better models by fixing that evaluation issues :) I think there are many other approaches to solve this classification problem and wanted to see how other people would do! 

Anyway about the evaluation issue, not sure I fully understand your point, but I guess it's the imbalance of the label that concerns you. In my own defense I'd say I used auc curve instead of accuracy in order to deal with the imbalance issue. (But I'm pretty sure the evaluation process could have been better-designed.) 

Thank you.
